src/inc/clr/fs/path.h

```diff
@@ -36,6 +36,25 @@ namespace clr
         class Path
         {
         public:
+#if !PLATFORM_UNIX
+            static const CHAR DirectorySeparatorChar = '\\';
```

Comment:
We already have two of these defined in palrt.h:
```C
#ifdef PLATFORM_UNIX
#define DIRECTORY_SEPARATOR_CHAR_A '/'
#define DIRECTORY_SEPARATOR_CHAR_W W('/')
#define DIRECTORY_SEPARATOR_STR_W W("/")
#define PATH_SEPARATOR_CHAR_W W(':')
#define PATH_SEPARATOR_STR_W W(":")
#else // PLATFORM_UNIX
#define DIRECTORY_SEPARATOR_CHAR_A '\\'
#define DIRECTORY_SEPARATOR_CHAR_W W('\\')
#define DIRECTORY_SEPARATOR_STR_W W("\\")
#define PATH_SEPARATOR_CHAR_W W(';')
#define PATH_SEPARATOR_STR_W W(";")
#endif // PLATFORM_UNIX
```
And palclr.h:
```C
#define DIRECTORY_SEPARATOR_CHAR_A '\\'
#define DIRECTORY_SEPARATOR_CHAR_W W('\\')
#define DIRECTORY_SEPARATOR_STR_W W("\\")

#define PATH_SEPARATOR_CHAR_W W(';')
#define PATH_SEPARATOR_STR_W W(";")
```

So I'd suggest adding VOLUME_SEPARATOR_CHAR over there and using the existing constants.